src/manipulation.js

```diff
@@ -46,14 +46,14 @@ wrapMap.optgroup = wrapMap.option;
 wrapMap.tbody = wrapMap.tfoot = wrapMap.colgroup = wrapMap.caption = wrapMap.thead;
 wrapMap.th = wrapMap.td;
 
-// Manipulating tables requires a tbody
 function manipulationTarget( elem, content ) {
-	return jQuery.nodeName( elem, "table" ) &&
-		jQuery.nodeName( content.nodeType !== 11 ? content : content.firstChild, "tr" ) ?
+	if ( jQuery.nodeName( elem, "table" ) &&
+		jQuery.nodeName( content.nodeType !== 11 ? content : content.firstChild, "tr" ) ) {
 
-		elem.getElementsByTagName("tbody")[0] ||
-			elem.appendChild( elem.ownerDocument.createElement("tbody") ) :
-		elem;
+		return elem.getElementsByTagName( "tbody" )[ 0 ] || elem;
```

Comment:
I will complete my previous comment there.

Basically, you can't use `innerHTML` property on the `table` element (only on `tbody`) in IE6-9 (i suppose at one point `innerHTML` might have been used like that), plus, in IE6-7 it would protect you from the case like this - 
```js
$( "<table/>" ).append( "<tr>...</tr>" );
```
No need for that now.

> Is it true for Android 2.x for example?

There could be situation when table exist with rows and columns without `tbody` parent - when you use DOM-manipulation methods. But it doesn't matter, since table should be rendered correctly with or without `tbody` in modern browsers. 

Search for `tbody` though, still necessary, since if user created table through html serialization (like through `innerHTML` property or defining it directly in html) will create `tbody` element even if it was omitted, so in order to correctly and consistently manipulate that table, we still need to search for it. 

And yeah, i checked it in Android 2.3... and IE6-7, Opera 9, Firefox 3.6 (Just to understand genesis of that code-path). They all follow that logic.